src/features/user/TreeMapper/index.tsx

```diff
@@ -40,22 +40,19 @@ function TreeMapper(): ReactElement {
     PlantLocationSingle | PlantLocationMulti | null
   >(null);
   const [links, setLinks] = React.useState<Links>();
-  const { tenantConfig } = useTenant();
   const { redirect, setErrors } = React.useContext(ErrorHandlingContext);
   async function fetchTreemapperData(next = false) {
     setIsDataLoading(true);
     setProgress(70);
 
     if (next && links?.next) {
       try {
-        const response =
-          await getAuthenticatedRequest<ExtendedScopePlantLocations>({
-            tenant: tenantConfig?.id,
-            url: links.next,
-            token,
-            logoutUser,
+        const response = await getApiAuthenticated<ExtendedScopePlantLocations>(
+          links.next,
+          {
             version: '1.0.4',
-          });
+          }
+        );
```

Comment:
Is version needed? Confirm and remove if possible.

Reply:
@sunilsabatp Once this point is addressed, this PR can be merged.

Reply:
I’ve confirmed with Jorgo that the backend sets a default value for version, so we can safely remove it from the frontend.